doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -50,16 +119,145 @@ to keep your client decoupled from implementation details.
+(<code>'</code>), e.g: <code>color=='light,green','deep,blue'</code>.</p>
+
+<p>Unary statements are composed of a prefix (either <code>+</code>or <code>-</code>) and are used in two situations:</p>
+
+<ul>
+  <li>To mean existence or no-existence of attributes. E.g. <code>+temperature</code>matches entities which have
+  a temperature attribute (no matter its value), while <code>-temperature</code>matches entities which don'e have
+  a temperature attribute.</li>
+  <li>To mean existence or no-existence of entity type, with the <code>type</code>keyword. E.g, <code>-type</code>
+  matches entities which don't have a type.</li>
+</ul>
+
+</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">geometry</td><td>
+
+<p>Defines a geografical area so only the entities located in that area matches the query. It is composed of list of
+tokens separated by <code>;</code>. The first token is the shape of the geometry, the rest of the tokens (if any) depends
+of the shape. In particular, the following geometries can be defined so far:</p>
+
+<ul>
+  <li><code>geometry=circle;radius:4000</code>, inside a circle (radius is in meters).</li>
+  <li><code>geometry=circle;radius:4000;external</code>, outside a circle (radius is in meters).</li>
+  <li><code>geometry=polygon</code>, inside a polygon.</li>
+  <li><code>geometry=polygon;external</code>, outside a polygon.</li>
+</ul>
+
+</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">coords</td><td>
+
+<p>List of coordinates (separated by <code>;</code>) which interpretation, depending on the geometry, is as follows:</p>
+
+<ul>
+  <li>In the case of <code>circle</code>geometry, the parameter includes just one element, which is the center of the circle.</li>
+  <li>In the case of <code>polygon</code>includes a list of elements, which are the vertices of the polygon.</li>
+</ul>
+
+</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">attrs</td><td>List of comma separated attributes to include in the entities of the response. If this parameter is not included, all the attributes are retrieved.</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">limit</td><td>Limit the number of entities to be retrieved</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">offset</td><td>Skip a number of records</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">options</td><td>
+This operation supports two options:
+
+<ul>
+  <li><code>count</code>: when used, the total entities number is returned as a HTTP in the response name <code>X-Total-Count</code>.</li>
+  <li><code>cannonical</code>: when used, the response payload used the "cannonical form". See JSON entity representation format section
+  for details.</li>
+</td></tr>
+</table>
+
+The response payload is an Array which contains one object per matching entity. Each entity follows
+the JSON entity representation format (described in a section above).
```

Comment:
I don't have found any simple way of doing this, thus it is deferred for the future.

NTC (in the context of this PR)